.travis.yml

```diff
@@ -1,16 +1,21 @@
 language: clojure
-lein: 2.8.1
-sudo: false
+sudo: required
 cache:
   directories:
     - $HOME/.m2
-script: lein do clean, test
+    - /usr/local
+install:
+  - sudo ./scripts/install-ci-deps /usr/local
+  - clojure -Sdescribe
+  - lumo --version
+script:
+  - ./scripts/test-jvm
+  - ./scripts/test-lumo
 branches:
   only:
     - master
 jdk:
-  - openjdk7
-  - oraclejdk7
+  - openjdk8
```

Comment:
Seems to me you should add here JDK 9 and 10 as well.

Reply:
Can do that